@types/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // tool-bar 1.2.x
 // https://atom.io/packages/tool-bar
+// Definitions by: aminya <https://github.com/aminya>
 
 /// <reference path="./config.d.ts" />
 
-import { TooltipManager } from "../index"
+import {TooltipPlacement, Disposable} from "atom" // "../index";
```

Comment:
Small nitpick, but I'd suggest removing this comment

```suggestion
import {TooltipPlacement, Disposable} from "atom";
```

Reply:
This is a reminder for creating PRs to the actual `@types/atom`. It should be `"../index"` for that repository. I prefer to keep it.